src/Roslyn.Diagnostics.Analyzers/UnitTests/DeclarePublicAPIAnalyzerTests.cs

```diff
@@ -142,7 +142,7 @@ public class C
                 GetCSharpResultAt(7, 43, DeclarePublicAPIAnalyzer.DeclareNewApiRule, "ArrowExpressionProperty.get"));
         }
 
-        [Fact(Skip = "821"), WorkItem(821, "https://github.com/dotnet/roslyn-analyzers/issues/821")]
+        [Fact, WorkItem(821, "https://github.com/dotnet/roslyn-analyzers/issues/821")]
```

Comment:
Can you also add a unit test that validates the contents of the public API file, probably just port the test added here: https://github.com/dotnet/roslyn-analyzers/pull/2036/files#diff-dd2170e9bef53f186020ef531e877547R206